### ISSUE TITLE
Exercise 9-5

### ISSUE THREAD

**Issue by bumzy** [posborne/linux-programming-interface-exercises#10]
Hello, your ideas have impressed and helped me a lot. Thanks!
[Exercise 9-5 answer](https://github.com/posborne/linux-programming-interface-exercises/tree/master/09-process-credentials#exercise-9-5) may have a mistake. 

```
// setuid()
/* suspend not possible */
/* resume not possible */
setuid(getuid());  /* Permanently drop privileges*/
```

**Comment by SteveLauC** [comment]
Indeed, for a `set-UID-root` program, `setuid(uid)` with a nonzero `uid` can permanently drop privilege as this will set `RUID/EUID/SUID` all to `uid`
